.storybook/preview.ts

```diff
@@ -3,6 +3,26 @@ import "../src/global.css";
 
 const preview: Preview = {
   parameters: {
+    /**
+     * @see https://storybook.js.org/docs/essentials/backgrounds
+     */
+    backgrounds: {
+      default: "Light",
+      values: [
+        {
+          name: "Light",
+          value: "#F3F6F4",
+        },
+        {
+          name: "Gray",
+          value: "#E5E4E2",
+        },
+        {
+          name: "Dark",
+          value: "#36454F",
+        },
+      ],
+    },
```

Comment:
I think there is a setup that allows us to change theme from light to dark that would be super useful.
This only changes the background but not the theme.

We need the theme change, but maybe we can also keep this as complement to that (if you see it makes sense).

https://storybook.js.org/recipes/tailwindcss#3-add-a-theme-switcher-tool

Reply:
The dark theme is out of scope, can be done in a new task (it would actually be better on another PR).

Reply:
Yes! Exactly, I wanted to change just the background for testing the popover without a border (and not lose it against a white bg).

I'd like to add the theme switch, but would like to do this in another  PR if it's possible, as it will affect also the component styling for all the stories

Reply:
As we are adding theme switcher,  do we need to keep this? 

Seem that serves the same purpose? what utility do you see it?

Reply:
okay, tried, think it's cool to change backgrounds, might be good to notice little changes.